### ISSUE TITLE
Allows users to customize which properties are watched.

### ISSUE THREAD

**Issue by cary-rowen** [cary-rowen/objWatcher#6]
Add a feature to the settings panel that allows users to select which properties to watch using a selectable list control, such as name, value, or description.

cc @hwf1324 
Would you like to implement this feature in the future?

**Comment by hwf1324** [comment]
Yes, I will consider implementing it. But not sure exactly when. Anyone interested in discussing this feature is welcome

**Comment by cary-rowen** [comment]
I'm implementing an extensive refactor, but no changes to the settings panel are expected in the next release.
